src/SFML/Window/Unix/KeySymToKeyMapping.hpp

```diff
@@ -1,7 +1,7 @@
 ////////////////////////////////////////////////////////////
 //
 // SFML - Simple and Fast Multimedia Library
-// Copyright (C) 2007-2021 Laurent Gomila (laurent@sfml-dev.org)
+// Copyright (C) 2007-2023 Laurent Gomila (laurent@sfml-dev.org)
```

Comment:
Not to forget that this can be updated on 2.6.x too.

Reply:
Done with #2376